README.md

```diff
@@ -81,6 +84,26 @@ to get document access.
 - [bit.ly/kueue-controller-design](https://bit.ly/kueue-controller-design)
 presents the detailed design of the controller.
 
+## Roadmap
+
+This is a high-level overview of the main priorities for 2023, in expected order of release:
```

Comment:
Maybe for a separate PR, but we should have a summary of existing features (a more extended version of what's mentioned in line 16).

Reply:
This worths a separate PR. Maybe together with v0.3.0.

Reply:
Yes :)